Assets/Scripts/Agents/AgentBase.cs

```diff
@@ -76,6 +79,17 @@ private void Awake() {
 
   // TODO(titan): Methods for updating the agent and terminating the agent.
 
+  public float MaxForwardAcceleration() {
+    return StaticConfig.AccelerationConfig?.MaxForwardAcceleration ?? 0;
+  }
+
+  public float MaxNormalAcceleration() {
+    float maxReferenceNormalAcceleration =
+        (StaticConfig.AccelerationConfig?.MaxReferenceNormalAcceleration ?? 0) * Constants.kGravity;
+    float referenceSpeed = StaticConfig.AccelerationConfig?.ReferenceSpeed ?? 1;
+    return Mathf.Pow(Speed / referenceSpeed, 2) * maxReferenceNormalAcceleration;
```

Comment:
Division by zero if `referenceSpeed` is zero.

Reply:
I'd argue that an exception should be thrown here because at a reference speed of 0, no normal acceleration should be possible.